### PR TITLE
Update Label Annotations section

### LINKED ISSUE #36655: Label `node.kubernetes.io/exclude-from-external-load-balancers` not registered / documented
**This is a Bug Report**

**Problem:**
I have a node in Minikube that's labelled with the key `node.kubernetes.io/exclude-from-external-load-balancers`.

This looks intended, but https://kubernetes.io/docs/reference/labels-annotations-taints/ doesn't document that label.

**Proposed Solution:**
Document that label; as a side effect, this registers it.

I can guess what this label does but I don't actually *know*. If you pick this issue up, part of your task is to find out that detail.

**Page to Update:**
https://kubernetes.io/docs/reference/labels-annotations-taints/

**Additional Information:**
/sig node
/language en
/priority backlog

PS. If this label isn't supposed to be used any more, we should also mark it as deprecated.

### PR DESCRIPTION
**Pull request by PriyanshuAhlawat**
This pr adds the description of `node.kubernetes.io/exclude-from-external-load-balancers` annotation.
closes #36655

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -159,6 +159,17 @@ Used on: ServiceAccount
...
 The value for this annotation must be **true** to take effect. This annotation indicates that pods running as this service account may only reference Secret API objects specified in the service account's `secrets` field.
 
+### node.kubernetes.io/exclude-from-external-load-balancer
+
+Example: `node.kubernetes.io/exclude-from-external-load-balancer`
+
+Used on: Node
+
+Kubernetes automatically enables the `ServiceNodeExclusion` feature gate on the clusters it creates. With this feature gate enabled on a cluster,
+you can add label to particular worker nodes to exclude them from the list of backend servers.
```
Comment (Michelle951) [inline content/en/docs/reference/labels-annotations-taints/_index.md:169]:
```suggestion
you can add labels to particular worker nodes to exclude them from the list of backend servers.
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -511,9 +522,9 @@ The kubelet checks D-value of the size of `/proc/sys/kernel/pid_max` and the PID
 
 Example: `node.kubernetes.io/out-of-service:NoExecute`
 
-A user can manually add the taint to a Node marking it out-of-service. If the `NodeOutOfServiceVolumeDetach` 
+A user can manually add the taint to a Node marking it out-of-service. If the `NodeOutOfServiceVolumeDetach`
 [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled on
-`kube-controller-manager`, and a Node is marked out-of-service with this taint, the pods on the node will be forcefully deleted if there are no matching tolerations on it and volume detach operations for the pods terminating on the node will happen immediately. This allows the Pods on the out-of-service node to recover quickly on a different node. 
+`kube-controller-manager`, and a Node is marked out-of-service with this taint, the pods on the node will be forcefully deleted if there are no matching tolerations on it and volume detach operations for the pods terminating on the node will happen immediately. This allows the Pods on the out-of-service node to recover quickly on a different node.
```
Comment (Michelle951) [inline content/en/docs/reference/labels-annotations-taints/_index.md:527]:
```suggestion
`kube-controller-manager`, and a Node is marked out-of-service with this taint, the pods on the node will be forcefully deleted if there are no matching tolerations on it and volume detach operations for the pods terminating on the node will happen immediately. This allows the pods on the out-of-service node to recover quickly on a different node.
```

**Review by sftim — COMMENTED** [review]
Thanks @PriyanshuAhlawat

It's better to have 1 commit that add `node.kubernetes.io/exclude-from-external-load-balancer` and a different commit (or different PR) to make the other changes that you're suggesting.

**Comment by PriyanshuAhlawat** [comment]
@sftim I only added the annotation section, I think the rest changes being shown are kind of automatically generated due to the insertion of this section.

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -645,7 +656,7 @@ This determines whether a user can modify the mode of the source volume when a
 {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}} is being
 created from a VolumeSnapshot.
 
-Refer to [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode) 
+Refer to [Converting the volume mode of a Snapshot](/docs/concepts/storage/volume-snapshots/#convert-volume-mode)
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:659]: It looks like your text editor has removed some whitespace here. Watch out for this - tidying up like this is best done in its own commit.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/reference/labels-annotations-taints/_index.md:527]: We shouldn't revise this text _here_; it's not the focus of this PR.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 3 above)
Comment (PriyanshuAhlawat) [reply content/en/docs/reference/labels-annotations-taints/_index.md:659]: okay, so is there anything I can do regarding this?

**Comment by sftim** [comment]
Thanks @PriyanshuAhlawat - a useful improvement.
